File: java/UberVan.java
class UberVan extends UberAdvanced {
    public UberVan (Integer id, String license, Account driver, Integer passengers, String typeCarAccepted, String seatsMaterial) {
        super(id, license, driver, passengers, typeCarAccepted, seatsMaterial);
    }

    @Override
    public void setPassengers (Integer passengers) {
        if (passengers >= 6) {
            this.passengers = passengers;
        } else {
            System.out.println("Car Passengers must be 6 o more");
        }
    }
}

File: js/uberVan.js
class UberVan extends UberAdvanced {
    constructor (id, license, driver, passengers, typeCarAccepted, seatsMaterial) {
        super(id, license, driver, passengers, typeCarAccepted, seatsMaterial);
    }

    setPassengers (passengers) {
        if (passengers >= 6) {
            this._passengers = passengers;
        } else {
            console.log("Car Passengers Must be 6 or more")
        }
    }
}